ietf/templates/doc/document_draft.html

```diff
@@ -173,7 +173,7 @@
                     </td>
                 </tr>
             {% endif %}
-            {% if doc.stream_id == 'ietf' or doc.stream_id == 'ise' or doc.stream_id == 'irtf' or doc.stream_id == 'editorial' %}
+            {% if doc.stream_id == 'ietf' or doc.stream_id == 'ise' or doc.stream_id == 'irtf' or doc.stream_id == 'editorial' or doc.stream_id == 'iab' %}
                 <tr>
                     <td></td>
                     <th scope="row">
```

Comment:
At the next line (L180) switch on stream_id to say "IAB Shepherd" for iab stream otherwise "Doc Shepherd" please